kienmarkdo/uOBites
# Hash/salt/encrypt passwords

To test, you need to reset the database:
```
cd backend 
pip install bcrypt (or pip3)
cd database
python db.py (or python3)
```
password can be found in the `populate_db.sql` in the comment if you forgot your password. Also, be careful, I have updated the `populate_db.sql` file so make sure you're using the right email
<img width="1132" alt="Screenshot 2023-07-18 at 23 40 27" src="https://github.com/kienmarkdo/uOBites/assets/71091659/4ac76447-f801-447e-9c78-f45342ee74e5">
